readme.md

```diff
@@ -55,14 +63,15 @@ Hello, Node!
 * **Question:** What is the difference between these two methods?
 
 * **Your Answer:**
-
+One is asynchronous, the other isn't. 
```

Comment:
Which is which?

Reply:
Readfile is asynchronous; I've added that to the readme.